samples/bluetooth/hci_uart/boards/nucleo_l476rg.overlay

```diff
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/ {
+	chosen {
+		zephyr,bt-c2h-uart = &usart2;
+	};
+};
```

Comment:
I don't think you should target a specific board but prefer work at shield level and use arduino label..

Alternatively provide these configurations directly at board level.

Reply:
Thanks for the comment. Fixed.